build-scripts/babel-plugins/custom-polyfill-plugin.js

```diff
@@ -2,6 +2,18 @@ import defineProvider from "@babel/helper-define-polyfill-provider";
 
 // List of polyfill keys with supported browser targets for the functionality
 const PolyfillSupport = {
+  // Note states and shadowRoot properties should be supported.
+  "element-internals": {
+    android: 90,
+    chrome: 90,
+    edge: 90,
+    firefox: 126,
+    ios: 17.4,
+    opera: 76,
+    opera_mobile: 64,
```

Comment:
```suggestion
    opera_mobile: 80,
```

Reply:
I'm guessing you got 80 from CanIUse?  That's the current version.  CanIUse does not keep historical data for mobile except for iOS and pre-evergreen.  Use MDN.